src/adu-shell/src/script_tasks.cpp

```diff
@@ -59,10 +59,23 @@ ADUShellTaskResult Execute(const ADUShell_LaunchArguments& launchArgs)
             {
                 filePermissionsChanged = true;
                 stat(path, &st);
-                Log_Warn(
+                Log_Error(
```

Comment:
Should it set error value (non-zero) using taskResult.SetExitStatus and essentially "goto done" (put done: label around line 92 before return) so it does not run any more logic?